src/tests/conformance/query/get_by_hash.rs

```diff
@@ -74,3 +75,49 @@ async fn should_get_transaction_by_hash() {
     assert!(synth_node.expect_message(&check).await);
     node.stop().expect("unable to stop stateful node");
 }
+
+#[tokio::test]
+async fn should_query_for_transactions_after_have_transactions() {
+    // ZG-CONFORMANCE-009
+    // Create stateful node.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = build_stateful_builder(target.path().to_path_buf())
+        .expect("unable to get stateful builder")
+        .start(false)
+        .await
+        .expect("unable to start stateful node");
+
+    // Wait for correct state and account data.
+    wait_for_state("proposing".into()).await;
+    let account_data = wait_for_account_data("rNGknFCRBZguXcPqC63k6xTZnonSe6ZuWt") // TODO make constant
```

Comment:
does this value come from the saved full node state? in that case it should rather be saved (alongside some other useful values like it) in some dedicated file during the process of producing a node's state, and loaded together with it; it can be done as a follow-up

Reply:
Transaction hash come from ledger, but it may be different for every setup (i.e. I have always the same, you will have always the same, but different). It can be extracted during setup and used later, but:
1. Makes setup a bit more difficult.
2. I wonder if we would have to wait for something else.

Reply:
Added TODO.

Reply:
yeah, it will add an extra step to the ledger dump, but it doesn't need to complicate the node setup - each hash and other element that we need could be loaded via
```
const THING: &str = include_str!("<ledger_location>/thing");
```